jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ResourceSelector.kt

```diff
@@ -138,7 +139,7 @@ class ResourceSelector<T> : ComboBox<T>() {
         }
     }
 
-    fun values(): Set<T> = (0 until itemCount).map { getItemAt(it) }.toSet()
+    fun values(): List<T> = (0 until itemCount).map { getItemAt(it) }.toList()
```

Comment:
Why is this changed?

Reply:
This is a leftover change suggested by @kiiadi in a previous PR. Returning list from here will support a caller who is interested in the order of items in future.

Reply:
Okay... I would have just changed the comnbo box model to CollectionComboBoxModel and used `CollectionComboBoxModel#getList()`

Reply:
^^ what was the resolution here? the main use of the `values` method is to perform an 'exists' function right? that's why `Set` is a better choice of data-structure - why should the caller ever need to know about the order of this? That's really a view (and thus the `model`'s) concern.

Reply:
The current caller doesn't need to know the order of elements, so Set is better. I used List initially but changed it to Set upon your suggestion in #899 where later you suggested that `the values() method should still return a list - other resource selectors may care about ordering (and may contain duplicates) which you will lose if you move this to a Set.`

Well, in the latest revision, I moved from using `ResourceSelector<String>` to `ResourceSelector<Stack>` where Stack is a data class containing stack name and id. Now I use `values()` method not to perform an 'exists' function but to find an element matching a condition, so Set doesn't give me any advantage.

I think I'll just move to CollectionComboBoxModel as Austin suggested and get rid of this method.

Reply:
you should be able to use `software.aws.toolkits.jetbrains.utils.ui.UiUtils#find` that allows you to find an item within a combobox model ...

Reply:
Thanks, I could use UiUtils#find. Removed the `values()` method from ResourceSelector as I don't need it now.